features.json

```diff
@@ -5,6 +5,7 @@
     "ember-libraries-isregistered": null,
     "ember-application-engines": true,
     "ember-glimmer": null,
+    "ember-glimmer-allow-two-way-reflush": true,
```

Comment:
New features generally land as `false` initially, so we can confirm that both sides of the feature flagging works.  I suspect that this feature should actually be a development only feature (ala `mandatory-setter`), which is setup in `ember-cli-build.js` ([here](https://github.com/emberjs/ember.js/blob/master/ember-cli-build.js#L27)).